dbfit-java/core/src/main/java/dbfit/api/AbstractDbEnvironment.java

```diff
@@ -126,14 +127,26 @@ public DdlStatementExecution createDdlStatementExecution(String ddl)
         return new DdlStatementExecution(getConnection().createStatement(), ddl);
     }
 
+    /**
+     * Create a procedure statement execution object for the given prepared statement
+     */
+    protected StatementExecution createStatementExecution(PreparedStatement statement) {
+        return new StatementExecution(statement);
+    }
+
     @Override
-    public StatementExecution createStatementExecution(PreparedStatement statement, boolean clearParameters) {
-        return new StatementExecution(statement, clearParameters);
+    public final StatementExecution createStatementExecution(String commandText) throws SQLException {
+        return createStatementExecution(getConnection().prepareStatement(commandText));
+    }
+
+    protected StatementExecution createCallExecution(PreparedStatement statement, boolean isFunction)
```

Comment:
`createCallExecution(PreparedStatement)` - it's used only in one place now and we can leave the `from-text` text version only but that makes it asymmetrical compared to `createStatementExecution`. I thought it would be cleaner if we follow the same pattern for both.